tests/foreman/maintain/test_remotedb.py

```diff
@@ -0,0 +1,87 @@
+"""Tests for Remote DB setup and upgrade
+
+:Requirement: foreman-maintain
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Foreman Maintain
+
+:Assignee: lvrtelov or vsedmik or gtalreja
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+
+from robottelo.config import settings
+from robottelo.logging import logger
+from robottelo.utils.issue_handlers import is_open
+
+
+@pytest.fixture(scope='module')
+def remotedb_setup(remotedb_sat):
+    installer = f"""satellite-installer --scenario satellite \
```

Comment:
You might want to use `InstallerCommand` class here.
https://github.com/SatelliteQE/robottelo/blob/5da310bbdb3332152879162f8989ec003f51f023/tests/foreman/installer/test_infoblox.py#L83

Reply:
Removed installer itself :)